src/lib.rs

```diff
@@ -81,10 +81,19 @@ unsafe extern "C" fn commit(
     txn: *mut pg::ReorderBufferTXN,
     _lsn: pg::XLogRecPtr,
 ) {
-    let s = CString::new("{ \"commit\": %u, \"t\": \"%s\" }").unwrap();
+    let j = r#"{"epoch": %u, "c_txid": %u, "commit": %u, "t": "%s", "xmin": %u, "c_xmin": %u}"#;
```

Comment:
So changing this breaks the test cases in `test/expected/*.out`. Since the format of the message has changed. We should update those to ensure this PR keeps passing tests.

Reply:
So it turns out, as long as we format our commit message almost the same, the tests just ignore commit calls, which makes sense, since they have timestamps and unstable commit ids... I think that is fine? 

Anyways, this is fixed